backend/curriculum_tracking/serializers.py

```diff
@@ -430,6 +441,22 @@ def get_cards_in_progress_column_as_assignee(self, user):
 
         return cards_in_progress_column_as_assignee
 
+    def get_cards_currently_blocked_as_assignee(self, user):
+
+        cards_currently_blocked_as_assignee = models.AgileCard.objects.filter(
+            Q(status=models.AgileCard.BLOCKED) & Q(assignees=user.id)
```

Comment:
This works, but why the added complexity? If you want to achieve an `and` then you can just chain multiple filters together (like you did with the other ones).

Q is useful for or, not and things that are more llke boolean algebra.  Always remember the principle of KISS.